website/static/js/osfHelpers.js

```diff
@@ -10,7 +10,7 @@ var iconmap = require('js/iconmap');
 
 // TODO: For some reason, this require is necessary for custom ko validators to work
```

Comment:
This line of code indicates that the original author wasn't entirely sure why something worked. 

Not in scope for this PR's timeline, but I try to at least *notice* such comments.